Movied/Movied/Scenes/Home/HomeViewController.swift

```diff
@@ -1,9 +1,215 @@
 import UIKit
 
 final class HomeViewController: UIViewController {
-
+    @IBOutlet private weak var homeTitle: UILabel!
+    @IBOutlet private weak var movieListCollectionView: UICollectionView!
+    @IBOutlet private weak var sortedByTextField: UITextField!
+    
+    private var allMovies: [Movie] = []
+    private var sortType: SortType = .newestToOldest
+    private var pageNumber = 1
+    
+    private let refreshControl = UIRefreshControl()
+    private let pickerView = UIPickerView()
+    private var refreshFooterView: RefreshFooterView?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        showIndicator(seconds: 2)
+        configView()
+        fetchMovies()
+    }
+    
+    //MARK: - Fetch Data
+    private func fetchMovies() {
```

Comment:
fetchMovies not clear mean because inside it set pageNumber to 1. can change to fetchFirstPage to clearer